.github/workflows/ci.yml

```diff
@@ -24,13 +24,20 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+
       - name: Setup .NET Core SDK ${{ matrix.dotnet-version }}
         uses: actions/setup-dotnet@v3
         with:
           dotnet-version: ${{ matrix.dotnet-version }}
+
+      - name: Build Nox.Generator.Tasks
+        run: dotnet build ./src/Nox.Generator.Tasks
+
```

Comment:
@rochar @jan-schutte 
The build is fixed now by adding an additional step to build Nox.Generator.Tasks project before restoring all the rest. The nuget package is created on this build and copied to ./src/nuget folder.